### DISCUSSION TITLE
Is it somehow possible to use oce for the Nortek Vectrino Profiler?

### DISCUSSION THREAD

**Discussion by JanDieterle** [dankelley/oce#1981] [Q&A]
Hello,  for my bachelor thesis about currents under floating photovoltaics I used the Nortek Vectrino Profiler. While I was looking for some ideas for the post processing of my data I saw your oce package. Unfortunately the package does not contain the Proflier but the Vector. Because I’m not that experienced with R I was wondering if you know if it’s easy or possible at all to convert the Profiler data shape somehow into the shape of the Vector data? So that I can use the oce for my data. 

Thanks for your help in advance.

**Comment by dankelley** [comment]
@richardsc may have insights on the various Nortek devices, but writing code for a new device is very time-consuming because the files have to be parsed in a byte-by-byte manner.  For example, the code to read AD2CP data (which is still only perhaps halfway functional after a *lot* of work) covers about 2000 lines of R and about 200 lines of C++.  The industry standard for coding used to be estimated as 10 lines of debugged code per day, so you can see that, even if R is 10X easier than the old languages, it might take 20 days to get code working.

Conclusion: if oce cannot read it at the moment, you ought to try to find another solution.  I think Nortek has software for sale, and also the IMOS toolbox (of matlab code) might be able to read your files.

**Reply by JanDieterle** [reply]
Alright, thanks for responding and your assessment. I'll try my best to find a solution.
